caps/amazon_search_local_capabilities.json

```diff
@@ -0,0 +1,36 @@
+{
+  "android": {
+    "adbExecTimeout": 45000,
+    "app": {
+      "local": "/Users/ishan.dhanwal/Downloads/Amazon India Shop, Pay, miniTV_22.14.2.350_Apkpure.apk"
```

Comment:
path should be dynamic, if I want to run the tests in my system then it will fail

Reply:
this app is not uploaded on a cloud, we have downloaded it from web for local execution, that is why local path is used